Project-1/README.md

```diff
@@ -63,60 +68,228 @@ T	T	F	T	T
 T	T	T	F	T
 T	T	T	T	T
 (d) is contignet.
+
 2. A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
+
   ```(a) (A→B)→C
   (b) (A→(B∨C))∨(C→¬A)
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
-``` 
+
+``` (a) (A ∨ C) ∧ (¬B ∨ C)
+(b) ¬A ∨ ¬B
+(c) (¬A∧¬B∧C)∨(¬A∧¬C)??
+used CNF calculator https://www.artima.com/cs/cnf.html & https://www.erpelstolz.at/gateway/formular-uk-zentral.html
+
 
 3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+
   ```(a)  B(x,y) that says that x is a brother of y
   (b)  A(x,y) that says that x is an aunt of y
   (c)  C(x,y) that says that x and y are cousins 
   (d)  O(x) that says that x is an only child
   (e)  T(x) that says that x has exactly two brothers 
-```
+
+```(a) ∃x∃y∃z(P(z,y)∧P(z,x)∧¬Fx) - There is some x and some y and some z such that z is the parent of y and z is the parent x and x is not female
+(b) ∃w∃x∃y∃z(P(z,y)∧P(w,z)∧P(w,x)∧Fx) - There is some w and some x and some y and some z such that z is the parent of y and w is the parent of z and w is the parent of x and x is female
+(c) ∃v∃w∃x∃y∃z(P(v,w)∧P(v,z)∧P(w,x)∧P(z,y)) - There is some v and some w and some x and some y and some z such that v is the parent of w and v is the parent of z and w is the parent of x and z is the parent of y
+(d) ∃x∃y∀z(P(y,z)→z=x) - There is some x and some y such that for all z if y is the parent of z then z and x are identical 
+(e) ∃v∃w∃x∃y∀z(P(v,x)∧P(v,w)∧¬Fw∧P(v,y)∧¬Fy∧∀z((P(v,z)∧¬Fz)→((z=x)∨(z=w)∨(z=y)))) - There is some v and some w and some x and some y such that for all z if v is the parent of x and v is the parent of w and w is not female and v is the parent of y and y is not female and for all instances of z v is a parent of z and z is not female ten z is identical with w or z is identical with w or z is identical with y
 
 4. Let V be a vocabulary of the attribute (concept) language with complements (ALC) consisting of a role name "parent_of" and a concept name "Male". Interpret parent_of as "x is a parent of y" and M as "x is male". Where possible define the following formulas in this vocabulary; where not possible, explain why: 
   ```(a)  B that says that x is a brother of y
   (b)  A that says that x is an aunt of y
   (c)  C that says that x and y are cousins 
   (d)  O that says that x is an only child
   (e)  T that says that x has exactly two brothers 
-```
```

Comment:
Karl I cannot get the coding to take in Github. So frustrating

Reply:
> Karl I cannot get the coding to take in Github. So frustrating

There is no spacing between "$" and symbols next to it. Maybe you could search for "Markdown and Latex"? That is weird because I copy your code and paste to my comments, and then it works!

Reply:
![image](https://user-images.githubusercontent.com/123985147/217654079-cd62857a-2642-4d81-979d-6470fbd558e5.png)

Reply:
I will try again!